sympy/physics/mechanics/kane.py

```diff
@@ -63,9 +76,39 @@ class KanesMethod(_Methods):
     are given in the explicit form by default. In other words, the kinematic
     mass matrix is $\mathbf{k_{k\dot{q}}} = \mathbf{I}$.
     In order to get the implicit form of those matrices/vectors, you can set the
-    ``explicit_kinematics`` attribute to ``False``. So $\mathbf{k_{k\dot{q}}}$ is not
-    necessarily an identity matrix. This can provide more compact equations for
-    non-simple kinematics (see #22626).
+    ``explicit_kinematics`` attribute to ``False``. So $\mathbf{k_{k\dot{q}}}$
+    is not necessarily an identity matrix. This can provide more compact
+    equations for non-simple kinematics (see #22626).
```

Comment:
Let's avoid sending users to the issues. I'd leave the issue references out.

Reply:
I just changed the spacing here, but I'll remove this reference and the reference I introduced myself on the other solvers.